projects/canopy/src/lib/side-nav/side-nav-bar-item/side-nav-bar-item.component.html

```diff
@@ -0,0 +1,9 @@
+<div class="lg-side-nav-bar-item__link" (click)="onClick()">
```

Comment:
Given this isn't a link, is the class name correct?

Reply:
Do you even need this, could you use host binding instead to listen to the event?

Reply:
I'd say the name can be updated.

Reply:
Yeah, the click can be moved to a hostlistener.